ts/src/client.ts

```diff
@@ -219,13 +224,13 @@ class GrpcClient<TRequest extends ProtobufMessage, TResponse extends ProtobufMes
     });
   }
 
-  rawOnError(code: Code, msg: string) {
+  rawOnError(code: Code, msg: string, trailers: Metadata = new Metadata()) {
```

Comment:
Aren't these headers and not trailers?

Reply:
Trailers in fact!

rawOnError calls the `onEndCallbacks` (https://github.com/improbable-eng/grpc-web/blob/master/ts/src/client.ts#L222)

Which has this signature:
`  onEndCallbacks: Array<(code: Code, message: string, trailers: Metadata) => void> = [];`
https://github.com/improbable-eng/grpc-web/blob/master/ts/src/client.ts#L44